pkg/attestation/attestation_test.go

```diff
@@ -0,0 +1,325 @@
+package attestation_test
+
+import (
+	_ "embed"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/blocky/nitrite/pkg/attestation"
+	"github.com/blocky/nitrite/pkg/nitrite_error"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed test/nitro-stag-attestation-doc.json
+var nitroStagAttestDocJSON string
```

Comment:
Currently unit testing against a nitro staging attestation. Should I test others (i.e., self-signed)?

Reply:
yes, we need to make sure that the self-signed attestation can be verified as well